src/player.js

```diff
@@ -1979,10 +1979,8 @@ export default class Player extends FakeEventTarget {
           }
           this._updateTracks(data.tracks);
           this.dispatchEvent(new FakeEvent(CustomEventType.TRACKS_CHANGED, {tracks: this._tracks}));
-          resetFlags();
         })
-        .catch(error => {
-          this.dispatchEvent(new FakeEvent(Html5EventType.ERROR, error));
+        .finally(() => {
```

Comment:
not supported in IE11, please verify we transpile this.
https://caniuse.com/promise-finally

Reply:
It's OK, we're using it with post-roll on ads controller as well, tested it as well on IE11